src/main/resources/rds-ca-2019-root.pem

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Should this be here?

Reply:
Yeah, it's the RDS certificate we need to connect to the database. It's publicly available and the same for everyone so it's not a secret or anything. This seems to be the least worst option. The other options are:
1. Get the code to download the certificate before it runs (not too bad but increases the code complexity and takes longer)
2. Build the lambda as a docker image (:hear_no_evil: )
I'm not averse to option 1 but this is simpler.

Reply:
It could go into tdr-configurations I think, we'd have to update the build process to copy it over before we do it.  I'll put a ticket in.  The certificate does get updated but that last one lasted for 4-5 years and they give you loads of notice before they're changed over because the old one stops working. I've added something to the README about it.